proposals/0004-haskell-org-repo-maintainers

```diff
@@ -0,0 +1,31 @@
+---
+author: Tom Ellis
+date-accepted: N/A
+discussion: https://github.com/haskell-org/committee/pull/11
+---
+
+# Write access to haskell.org website repo
+
+This policy determines who should have write access to any settings
+and source repositories associated with the www.haskell.org website.
+The settings and source repositories are currently hosted on the
+GitHub platform, at
+<https://github.com/haskell-infra/www.haskell.org/>.
```

Comment:
Maybe add a paragraph about even though we're limiting write access, we're happy to accept PRs etc.